package.json

```diff
@@ -33,9 +33,9 @@
     "size:debug": "npm run build && size-limit --why",
     "start": "npm run storybook",
     "std-version": "standard-version --releaseCommitMessageFormat \"chore(release): version {{currentTag}} build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
-    "storybook": "npm run build:storybook && start-storybook -p 12123 -s .storybook/static/,node_modules/@sap-theming",
+    "storybook": "npm run build:storybook && start-storybook -p 12123 -s .storybook/static/,node_modules/@sap-theming --no-dll",
     "storybook:ci": "npm run storybook -- --ci --quiet",
-    "storybook:static": "rimraf storybook-static && npm run build:storybook && STORYBOOK_ENV=docs build-storybook -c .storybook -o storybook-static -s .storybook/static/,node_modules/@sap-theming --docs",
+    "storybook:static": "rimraf storybook-static && npm run build:storybook && STORYBOOK_ENV=docs build-storybook -c .storybook -o storybook-static -s .storybook/static/,node_modules/@sap-theming --docs --no-dll",
```

Comment:
`--no-dll` solves the popper version issue https://github.com/storybookjs/storybook/blob/774517be784f59202f4f4c293b80826cacbb02bb/docs/workflows/faq.md#why-im-getting-errors-with-react-popper-and-storybook